core/docs/spark-qualification-tool.md

```diff
@@ -32,50 +20,70 @@ applicable environments.  Here are the cluster information for the ETL benchmark
+> used in the Spark application and currently supported by the RAPIDS Accelerator for Apache Spark. Not all expressions and datatypes are currently supported,
+> Please refer to the "[Understanding Execs report](#execs-report)" section and the
+> "[Supported Operators](https://github.com/NVIDIA/spark-rapids/blob/main/docs/supported_ops.md)" guide to verify the types and expressions used are supported.
+
+## Table of Contents
+
+- [Running the Qualification Tool](#Running-the-Qualification-Tool)
+   - [Standalone on Spark event logs for CSP environments](#running-the-standalone-qualification-tool-for-csp-environments-on-spark-event-logs)
+   - [Standalone on Spark event logs for other environments](#running-the-qualification-tool-standalone-on-spark-event-logs)
+   - [Inside a running Spark application using the API](#running-the-qualification-tool-inside-a-running-spark-application-using-the-api)
+   - [On active jobs using a Spark Listener](#running-the-qualification-tool-using-a-spark-listener)
+- [Understanding the Qualification Tool Output](#understanding-the-qualification-tool-output)
+   - [Application Report](#application-report)
+   - [Stages Report](#stages-report)
+   - [Execs Report](#execs-report)
+   - [MLFunctions Report](#mlfunctions-report)
+   - [MLFunctions Total Duration Report](#mlfunctions-total-duration-report)
+   - [HTML Report](#html-report)
+- [Example Reports and File Output](#example-reports-and-file-output)
+- [Compiling the Tools Jar from Source](#compiling-the-tools-jar-from-source)
+- [S3 File Path Support and Configuration](#s3-file-path-support-and-configuration)
+- [Tools Jar Maven Dependency](#qualification-tool-maven-dependency-snippet)
 
-This document covers below topics:
 
-* TOC
-{:toc}
 
-## How to use the Qualification tool
+## Running the Qualification Tool
 
-The Qualification tool can be run in three different ways. One is to run it as a standalone tool on the
-Spark event logs after the application(s) have run, the second is to be integrated into a running Spark
-application using explicit API calls, and the third is to install a Spark listener which can output
-results on a per SQL query basis.
+The Qualification tool can be run three different ways:
+1. A standalone application running on Spark event logs after the applications are done
+2. Integrated into a running Spark application using explicit API calls
+3. Installing a Spark listener to output results on a per SQL query basis
 
-In running the qualification tool standalone on Spark event logs, the tool can be run as a user tool command
-via a [pip package](https://pypi.org/project/spark-rapids-user-tools/) for CSP environments (Google Dataproc,
-AWS EMR, Databricks AWS) or as a java application for other environments.
+Standalone mode can be run as a java application or using the `spark_rapids_user_tools` application that is available as the `spark-rapids-user-tools` [pip package](https://pypi.org/project/spark-rapids-user-tools/).
```

Comment:
I think we need to give more feasibility to `spark_rapids_user_tools`. I suggest we put at the very top then lists the other options to run the qualification tool through java environment.